<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BT</class>
 <widget class="QMainWindow" name="BT">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algoritmo BT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>241</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>119</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nombre </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>num_TT</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>-10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Algoritmo BT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="reloj">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de corrida:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>431</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>429</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>300</number>
    </attribute>
    <column>
     <property name="text">
      <string>                           Testores Típicos                                  </string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de Testores Tipicos: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detener">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="porciento">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="check2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Detener cuando el k% de los rasgos pertenezcan a testores típicos</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numTT">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="check1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Detener luego de encontrar al menos k testores típicos</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="intervalo">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>13</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tiempo en segundos de actualizar el debuger</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
